bin/check-rabbitmq-consumers.rb

```diff
@@ -18,58 +18,29 @@
 # LICENSE:
 # Copyright 2014 Daniel Kerwin <d.kerwin@gini.net>
 # Copyright 2014 Tim Smith <tim@cozy.co>
+# Copyright 2018 Mike Murray <37150283+monkey670@users.noreply.github.com>
```

Comment:
`<37150283+monkey670@users.noreply.github.com>Just checking that you want ` and not `<monkey670+37150283@users.noreply.github.com>`

Reply:
I've just checked and this is the correct one listed 37150283+monkey670@users.noreply.github.com.